http.go

```diff
@@ -70,6 +70,10 @@ func (s *Server) ServeHTTPS() {
 		log.Fatalf("FATAL: loading tls config (%s, %s) failed - %s", s.Opts.TLSCertFile, s.Opts.TLSKeyFile, err)
 	}
 
+	if len(s.Opts.TLSClientCAFiles) > 0 {
+		config.ClientAuth = tls.RequestClientCert
```

Comment:
you actually need to build a cert pool from the CA files and set it as `config.ClientCAs`